src/gtest/test_bip32.cpp

```diff
@@ -36,6 +42,22 @@ struct TestVector {
     }
 };
 
+class TestBip32 : public Test
+{
+public:
+    void SetUp() override
+    {
+        SelectParams(CBaseChainParams::Network::MAIN);
+    }
+
+    void TearDown() override
+    {
+    }
+
+
+};
+
+
 TestVector test1 =
```

Comment:
This is classic candidate for parameterized test.
two tests with only one parameter (TestVector).
class PTestBip32 : public TestWithParam<TestVector>
{}
...
TEST_P instead of TEST_F
{
     const auto &test = GetParam();
    ..... whatever is in RunTest
}

INSTANTIATE_TEST_SUITE_P(bip32, PTestBip32, 
   Values(
        TestVector1,
        TestVector2
    ));

Reply:
Fixed. Thanks